src/app/contextBroker/contextBroker.cpp

```diff
@@ -325,6 +328,13 @@ PaArgument paArgs[] =
 
 
 //
+// /v2 conv-ops 
+//
+#define ENT                EntitiesRequest
+#define ENT_COMPS_V2       2, { "v2", "entities" }
```

Comment:
probably we can define a constant for 'entities' and use it as well in the entryPoints operation as well, WDYT?